scripts/ci/run_tests.sh

```diff
@@ -7,4 +7,4 @@ fi
 
 echo "Using ${HADOOP_DISTRO} distribution of Hadoop from ${HADOOP_HOME}"
 
-python test/runtests.py -v
+python test/runtests.py -v $@
```

Comment:
Yes, you're right. I thought that run_tests.sh is only used by Travis. LGTM. Sorry about the delay.